js/main.js

```diff
@@ -1,2 +1,32 @@
 /* global data */
 /* exported data */
+var $photoUrl = document.querySelector('#photo-url');
+var $previewImg = document.querySelector('#preview-img');
+
+function addPhoto(event) {
+  $previewImg.setAttribute('src', $photoUrl.value);
+}
+
+$photoUrl.addEventListener('input', addPhoto);
+
+var $entryForm = document.querySelector('form');
+
+function formSubmit(event) {
+  event.preventDefault();
+  var formValues = {};
+
+  formValues.title = $entryForm.elements.title.value;
+  formValues.photoUrl = $entryForm.elements.photoUrl.value;
+  formValues.notes = $entryForm.elements.notes.value;
+
+  formValues.entryId = data.nextEntryId;
+  data.nextEntryId++;
```

Comment:
You can do both of these one one line if you do `formValues.entryId = data.nextEntryId++;` 🤓